bindings/nodejs/examples/how_tos/nfts/burn_nft.ts

```diff
@@ -24,42 +24,38 @@ async function run() {
             storagePath: process.env.WALLET_DB_PATH,
         });
 
-        // Get the account we generated with `01-create-wallet`
-        const account = await wallet.getAccount('Alice');
-
+        // Get the wallet we generated with `01-create-wallet`
```

Comment:
why comment here?

Reply:
Was just updating the old comment, but doesn't make any sense anymore here :+1: